src/view/commands/submit_request.rs

```diff
@@ -126,10 +138,10 @@ where
                 .collect()
         };
 
-        self.writer_stderr.print_lines_styled([title_status]);
-        self.writer_stderr.print_lines([BREAK_LINE_WITH_GAP]);
-        self.writer_stderr.print_lines_styled(headers);
-        self.writer_stderr.print_lines([BREAK_LINE_WITH_GAP]);
+        self.writer_stdout.print_lines_styled([title_status]);
+        self.writer_stdout.print_lines([BREAK_LINE_WITH_GAP]);
+        self.writer_stdout.print_lines_styled(headers);
+        self.writer_stdout.print_lines([BREAK_LINE_WITH_GAP]);
```

Comment:
The use of "stderr" here is intentional. If user wants to pipe or redirect the output payload of request, he can use  the stdout operator. Like `treq GET url.com 1> output.json`, will only write int file the body of response.

Reply:
so the output payload of request is passed to stdout while the other metadata of the request is passed to stderr

Reply:
Exactly